### PR TITLE
Update the circle ci node pipeline

### PR DESCRIPTION
**Pull request by hopemanryan**
https://rookout.atlassian.net/browse/RK-13765

Issue with missing folder was fixed 
file fetching of docusarus has changed, so I had to hack around it 

changed the swagger url change process to node (Dont know why it failed all of sudden but now it works for sure)

Most file changes are due to yarn run build (docusaurus build ) and its internal post install cmd

### CODE REVIEW

File: website/.docusaurus/registry.js [hunk 1]
```diff
@@ -1,55 +1,48 @@
...
-  'versionMetadata---docs-935-398': [() => import(/* webpackChunkName: 'versionMetadata---docs-935-398' */ '~docs/default/version-current-metadata-prop-751.json'), '~docs/default/version-current-metadata-prop-751.json', require.resolveWeak('~docs/default/version-current-metadata-prop-751.json')],};
+  '013afb70': [() => import(/* webpackChunkName: '013afb70' */ '/Users/ryan/Desktop/projects/docs/website/.docusaurus/docusaurus-plugin-content-docs/default/plugin-route-context-module-100.json'), '/Users/ryan/Desktop/projects/docs/website/.docusaurus/docusaurus-plugin-content-docs/default/plugin-route-context-module-100.json', require.resolveWeak('/Users/ryan/Desktop/projects/docs/website/.docusaurus/docusaurus-plugin-content-docs/default/plugin-route-context-module-100.json')],
```
Comment (Urook) [inline website/.docusaurus/registry.js:2]: Will this work with that path?

File: website/build_utils/fix_swagger_url.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+const fs = require('fs');
+const path = require('path');
+const process = require('process');
+const defaultSwaggerUrl = "https://petstore.swagger.io/v2/swagger.json"
+const rookOutSwaggerUrl = "https://app.rookout.com/api/v1/rookout.yaml"
```
Comment (Urook) [inline website/build_utils/fix_swagger_url.js:5]: Can you add the v2 yaml as well?

**Review by Urook — CHANGES_REQUESTED** [review]

File: website/.docusaurus/registry.js (hunk 1 above)
Comment (hopemanryan) [reply website/.docusaurus/registry.js:2]: I mean this self generated

**Comment by Urook** [comment]
@hopemanryan I don't understand wjat most of the changes mean😅

**Comment by hopemanryan** [comment]
@Urook most of the changes here are auto generated from docusarus build process

File: website/build_utils/fix_swagger_url.js (hunk 2 above)
Comment (hopemanryan) [reply website/build_utils/fix_swagger_url.js:5]: instead of v1 ?

Comment (Urook) [reply website/build_utils/fix_swagger_url.js:5]: No, in addition (If only one is possible, have v1)

**Comment by Urook** [comment]
> @Urook most of the changes here are auto generated from docusarus build process 

Can you please write this command in the description?

File: website/.docusaurus/registry.js (hunk 1 above)
Comment (Urook) [reply website/.docusaurus/registry.js:2]: Hmmm. This is a public repo. Any way to generate it with nutral paths?

File: website/build_utils/fix_swagger_url.js (hunk 2 above)
Comment (hopemanryan) [reply website/build_utils/fix_swagger_url.js]: Done

File: website/.docusaurus/registry.js (hunk 1 above)
Comment (hopemanryan) [reply website/.docusaurus/registry.js]: removed the folder completely, as its recreated in build time

**Comment by Urook** [comment]
@hopemanryan Why is this called upgrade node version? Also, it looks fine to me, but I am not very familiar with the docs project. Are you sure you don't want someone else looking at this too? If not, I'll approve, just change the title

File: website/.docusaurus/DONT-EDIT-THIS-FOLDER [hunk 3]
```diff
@@ -1,5 +0,0 @@
-This folder stores temp files that Docusaurus' client bundler accesses.
-
-DO NOT hand-modify files in this folder because they will be overwritten in the
```
Comment (Urook) [inline website/.docusaurus/DONT-EDIT-THIS-FOLDER:3]: Please add the .docusaurus dir to .gitignore (unless we need some of its files in the git index

**Review by Urook — CHANGES_REQUESTED** [review]

**Comment by hopemanryan** [comment]
> @hopemanryan Why is this called upgrade node version? Also, it looks fine to me, but I am not very familiar with the docs project. Are you sure you don't want someone else looking at this too? If not, I'll approve, just change the title and do the gitignore thing


Tbh this is the process of updating the build pipeline which started with node version 

Things should work now

**Comment by Urook** [comment]
> > @hopemanryan Why is this called upgrade node version? Also, it looks fine to me, but I am not very familiar with the docs project. Are you sure you don't want someone else looking at this too? If not, I'll approve, just change the title and do the gitignore thing
> 
> Tbh this is the process of updating the build pipeline which started with node version
> 
> Things should work now

Then perhaps something like "Update the circle ci node pipeline"?

File: website/.docusaurus/DONT-EDIT-THIS-FOLDER (hunk 3 above)
Comment (hopemanryan) [reply website/.docusaurus/DONT-EDIT-THIS-FOLDER:3]: done

**Review by Urook — APPROVED** [review]

**Pull request merged**